dot/parachain/overseer/overseer.go

```diff
@@ -270,13 +263,31 @@ func (o *OverseerSystem) handleBlockEvents() {
 		}
 	}
 }
-
 func (o *OverseerSystem) broadcast(msg any) {
 	for _, overseerToSubSystem := range o.subsystems {
 		overseerToSubSystem <- msg
 	}
 }
 
+func (o *OverseerSystem) headSupportsParachain(hash common.Hash) (bool, error) {
```

Comment:
how about `checkHeadSupportsParachainConsensus` or `DoesHeadSupportsParachainConsensus`?

Question sounding method names are sorta more apt for methods that return bool